package.json

```diff
@@ -43,6 +43,7 @@
     "deep-assign": "^2.0.0",
     "document-register-element": "dmarcos/document-register-element#8ccc532b7f3744be954574caf3072a5fd260ca90",
     "load-bmfont": "^1.2.3",
+    "lodash": "^4.17.15",
```

Comment:
use `deep-asign`. We can move to something else in the future if needed

Reply:
deep-assign fails to merge null values correctly. Thoughts?

Reply:
Avoid nulls or cycle through the object keys replacing nulls. lodash is a pretty big dependency.

Reply:
Will do. I think it doesn’t matter anyway if embedding Cordova checks is the preferred approach. Much less code changes. Will update this PR soon.